### ISSUE TITLE
Build-AUDIENCE:  checkbox label contains dash

### ISSUE THREAD

**Issue by XiuJuanXia** [mmiron-HC/WST-RSAMS#142]
URL: https://rsams.dev.cloud.openplus.ca/en/node/64019/edit?display=section_1&destination=/en/node/64019/build
User Agent: Chrome (v99), Firefox (v98)
Language: Both

Issue Description: In the "AUDIENCE", there are a list of checkboxes. Some of the labels of the checkboxes starts with a dash.
![image](https://user-images.githubusercontent.com/88776404/160846479-616b8ec3-86e9-41be-bd6b-cb4f4d8f8b42.png)

Steps to reproduce:
1. Login as an admin: https://rsams.dev.cloud.openplus.ca/en/user/login
2. Activate "Build"
3. Edit "Section 1 - Summary"
4. Navigate to "AUDIENCE", and label for checkboxes starts with a dash.

Expected results: Checkbox label should not start with a dash.

**Comment by echouin** [comment]
@XiuJuanXia this is working as intended. The - are sub-categories that we now have available to users.

**Comment by XiuJuanXia** [comment]
@echouin If the dash are sub-categories, this becomes a WCAG issue. Because nobody knows this is sub-category - neither sighted users nor non-sighted users. I updated the labels.

**Comment by davidgotlieb** [comment]
@XiuJuanXia Can I better understand why this is considered Critical failure? This seems like a pretty minor issue to me?

**Comment by XiuJuanXia** [comment]
@davidgotlieb It is critical as this isn't not being conveyed to anyone. If you feel this is a lower risk you can explain it when it goes to the CIO.

**Comment by davidgotlieb** [comment]
@XiuJuanXia @mmiron-HC But what's the impact of it? Shouldn't the severity be tied to how it would impact accessibility etc.? Isn't that the whole point of marking them with severity tags, so we can focus on the ones that will be a barrier to people using the site properly?

From a user perspective, they will check off the audiences that apply to the recall when doing data entry. Having a dash or not won't have any impact on their ability to do that, nor will it confuse them. So I just don't understand how this is a critical issue. To me this is at most moderate, when I compare it to some of the other items.

**Comment by mmiron-HC** [comment]
as per our discussion, While we, WST, have agreed that this is not considered critical, we disagree that this is lower than a serious. We have discussed this at length and have agreed to disagree.

**Comment by davidgotlieb** [comment]
Hi @mmiron-HC ,

Given that we don't have any development time right now, I asked OP if there were any quick configuration solutions to this issue.

They have changed the config in the dev environment to use dropdowns, which has a clearer delineation for the sub categories, but to be honest I think is an even worse user experience, and will also be much more time consuming to use.

Would something like this address your concern? Anything else would require them to do custom code.

![image](https://user-images.githubusercontent.com/53194802/163257064-d8114eb0-1a4e-47db-94bc-78cb007b03d7.png)

**Comment by mmiron-HC** [comment]
@davidgotlieb without really looking at it we can't tell 100% but we do feel just by looking at it that it isn't a great solution. We will have more time to look at it next week.

**Comment by davidgotlieb** [comment]
I've put this on the list of things we'll look at for the additional 2 sprints we've added, will see what can be done but I think this should be lower priority than some of the other serious items that have been flagged.

**Comment by davidgotlieb** [comment]
@mmiron-HC We're still struggling to find a fix for this that won't create bigger problems.

What if I reached out to the disability network and tried to get some staff with cognitive disabilities to help us with a usability test? If, theoretically they tell us it's not a big issue, would that satisfy?

Obviously if they struggle we'll have to keep working on it.

**Comment by davidgotlieb** [comment]
Issue reported as a Drupal core accessibility issue. Feature should be producing proper code with an nested unordered list so that hierarchy is clear.

Issue reported here: https://www.drupal.org/project/drupal/issues/3327459
